ai_bridge/models/ai_bridge.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import json
+
+from odoo import api, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class AiBridge(models.Model):
+
+    _name = "ai.bridge"
+    _description = "Ai Bridge Configuration"
+    _order = "sequence, id"
+
```

Comment:
It would be necessary to make it multi-company. Do you think it is correct for this version?